keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -386,18 +384,36 @@ logs or events for this purpose.
   The usage of this feature requires the manual step of applying a taint
   so the operator should be the one applying it.
 
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [X] API .status
+  If it works, pods in the stateful workload should be re-scheduled to another
+  running node. `Phase` in Pod `Status` should be `Running` for a new Pod
+  on the other running node.
+  If not, check the pod status to see why it does not come up.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
   - [X] Metrics
     - Metric name: 
-      - We can add new metrics `deleting_pods_total`, `deleting_pods_error_total`
-      in Pod GC Controller.
-      For Attach Detach Controller, there's already a metric:
-      attachdetach_controller_forced_detaches
-      It is also useful to know how many nodes have taints. We can explore with [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) which generates metrics about the state of the objects.
+      - New metrics are added in Pod GC Controller:
```

Comment:
Literally, `force_delete_pods_total`  and `force_delete_pod_errors_total`  metrics should count all the behaviors of the gc controller that forcefully delete pods.

In order to distinguish, the pod is deleted because of out-of-service, we should add a label to the metrics to identify the reason for the forced deletion.

I submit a pr.  https://github.com/kubernetes/kubernetes/pull/118480

Reply:
@msau42 @gnufied What do you think if we add this metric to all pods that are forcefully deleted by the Pod GC controller, not just with this out-of-service taint?

Reply:
I think that is reasonable. It will give more accurate picture of gc conroller.